### PR TITLE
feat: added java welcome-email example

### PR DESCRIPTION
**Pull request by christyjacob4**
* Java Welcome Email Example
* Rename deprecated `APPWRITE_FUNCTION_EVENT_PAYLOAD` to `APPWRITE_FUNCTION_EVENT_DATA`

### CODE REVIEW

File: java/welcome-email/src/main/java/WelcomeEmail.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    private static String YOUR_DOMAIN_NAME;
+    private static String API_KEY;
+
+    public static void main(String[] args) throws UnirestException {
+        YOUR_DOMAIN_NAME = System.getenv("MAILGUN_DOMAIN");
+        API_KEY = System.getenv("MAILGUN_API_KEY");
+        String payload = System.getenv("APPWRITE_FUNCTION_EVENT_DATA");
+
+//        System.out.println("[INFO] Payload");
+//        System.out.println(payload);
```
Comment (lohanidamodar) [inline java/welcome-email/src/main/java/WelcomeEmail.java:17]: do we need to keep these comments

**Review by lohanidamodar — COMMENTED** [review]
Simple query.
Rest LGTM.

File: java/welcome-email/src/main/java/WelcomeEmail.java (hunk 1 above)
Comment (christyjacob4) [reply java/welcome-email/src/main/java/WelcomeEmail.java:17]: I'll remove these :+1:

**Review by lohanidamodar — APPROVED** [review]

**Review by PineappleIOnic — APPROVED** [review]
Looks Good! :+1:

**Review by abnegate — APPROVED** [review]

**Pull request merged**